src/Libraries/Microsoft.Extensions.Http.Telemetry/Logging/Internal/OptionsExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.Http.Telemetry.Logging.Internal;
+
+internal static class OptionsExtensions
+{
+    public static TOptions GetKeyedOrCurrent<TOptions>(this IOptionsMonitor<TOptions> options, string? serviceKey)
```

Comment:
[nit] It'd be good to have xml-docs here too.

Reply:
Will add

Reply:
Done